docker-compose.dev.yml

```diff
@@ -11,7 +11,7 @@ services:
       target: node
     image: thenewboston-node:current
     ports:
-      - 127.0.0.1:8555:8555
+      - 127.0.0.1:8000:8555
```

Comment:
why?

Reply:
Because of the conflict between the `reverse-proxy` which exposes 8555 port and the `node` in dev config also exposes 8555 port and fails.